ps2.py

```diff
@@ -0,0 +1,13 @@
+'''Problem : Add two numbers
+Problem Statement
+Every problem starts with a Problem Statement. It tells you in detail about the task to be solved.
+Shivam is the youngest programmer in the world, he is just 12 years old. Shivam is learning programming and today he is writing his first program.
+The task is very simple: given two integers A and B, write a program to add these two numbers and output it.'''
+# Code :
+T = int(input())
+for tc in range(T):

```

Comment:
What Does this Loop Do?

Reply:
The loop deals with number of testcases . If the number of testcases is 3 then the loop iterates for 3 times.